velox/functions/prestosql/aggregates/tests/MaxSizeForStatsTest.cpp

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/functions/prestosql/aggregates/tests/AggregationTestBase.h"
+
+using namespace facebook::velox::exec::test;
+
+namespace facebook::velox::aggregate::test {
+
+namespace {
+
+class MaxSizeForStatsTest : public AggregationTestBase {
+ public:
+  void SetUp() override {
+    AggregationTestBase::SetUp();
+    disableSpill();
+  }
+};
+
+// Input:
+// c0(bigint)|c1(tinyint)
+// null|null
+// Query:
+// select aggr(c0),aggr(c1)
+// Output
+// null|null
+TEST_F(MaxSizeForStatsTest, nullValues) {
+  auto vectors = {makeRowVector({
+      makeNullableFlatVector(std::vector<std::optional<int64_t>>{std::nullopt}),
```

Comment:
This can be shortened to `makeNullableFlatVector<int64_t>({std::nullopt})`.

It would be nice to tests with more than one null row.